misc/ci_setup.py

```diff
@@ -255,23 +255,25 @@ def run(self):
             print('PYTHONPATH={}'.format(os.environ['PYTHONPATH']))
 
             execute_command('echo $PYTHONPATH')
-        elif get_os_name() != 'win':
+        else:
             # compile ..
             arg = environ.get('CI_SETUP_CMAKE_ARGS', '')
             os.makedirs('build', exist_ok=True)
-            execute_command(
-                f'TAICHI_CMAKE_ARGS="{arg}" {sys.executable} setup.py install --user'
-            )
+            if get_os_name() != 'win':
```

Comment:
set `TAICHI_CMAKE_ARGS={args}` as a new env variable, then we can unify this part?